app/views/idv/doc_auth/document_capture.html.erb

```diff
@@ -14,7 +14,7 @@
     liveness: liveness_checking_enabled?.presence,
     endpoint: api_verify_images_url
   } %>
-  <noscript>
+  <div class="js-fallback">
```

Comment:
Is changing this from `<noscript>` to `<div>` going to bring back a flicker? That was the reason we added the content there, right?

Reply:
Yes, it might. At least as long as it takes [this snippet of JavaScript](https://github.com/18F/identity-idp/blob/ab6fbe049eea6b885e35ad81411e21b7a1392f7a/app/javascript/app/utils/index.js#L10) to load and be executed.

I was having some trouble with `noscript` here, some combination of JavaScript not having access to pieces within the `noscript` and/or `input` fields not being considered by the server if JavaScript is enabled (error about required fields). I'll check again in my morning and see if there's any way to try to restore `noscript` behavior.